API.md

```diff
@@ -0,0 +1,50 @@
+# GPIO Expander Pi API
+
+## `class GpioExpander`
+
+Create an object `GpioExpander` to communicate with a particular board based on Amperka GPIO expander. The end-user libraries often wrap (or inherit) this class to provide domain-specific high level hardware interaction methods.
+
+### `GpioExpander(i2c_address=42)`
+
+Constructor. The `i2c_address` specifies the board I²C address which is `42` factory-default but can be changed programmatically.
+
+### `pin_mode(pin: int, mode: PinMode) -> None`
+
+Configures the operation `mode` of the `pin`:
+
+- PinMode.INPUT
+- PinMode.INPUT_PULLUP
+- PinMode.INPUT_PULLDOWN
+- PinMode.OUTPUT
+
+### `analog_read(pin: int) -> float`
+
+Reads the analog value from one pin of board. Return value in range from `0` to `1.0`.
+
+### `digital_read(pin: int) -> bool`
+
+Reads the bit value from one pin of board. Return `GPIO.HIGH` or `GPIO.LOW` value. This values defined as `True` and `False` respectively.
```

Comment:
```suggestion
Reads the bit value from one pin of the board. Return `GPIO.HIGH` or `GPIO.LOW` value. These values defined as `True` and `False` respectively.
```

So… why `GPIO.HIGH` and `GPIO.LOW` are ever required? Why can’t they be just `True` or `False`. OK, Arduino API does use HIGH and LOW, but this is _not_ Arduino and the de-facto standard `gpiozero` library does not use HIGH/LOW, etc.

Suggest KISS: just use `bool`.

Reply:
Done